packages/graphql-platform-connector-mariadb/src/graphql-platform/connector/database/operation/find.ts

```diff
@@ -209,7 +209,7 @@ export class FindOperation extends AbstractOperationResolver<
             case 'contains':
               value !== null && (
                 column.fullText !== null && value.length >= column.fullText
-                  ? where.addRaw(`MATCH(${column.name}) AGAINST('${value}' IN NATURAL LANGUAGE MODE)`)
+                  ? where.addRaw(`MATCH(${column.name}) AGAINST('${value}' IN NATURAL BOOLEAN MODE)`)
```

Comment:
It's not the right syntax -> `IN BOOLEAN MODE`

cf: https://mariadb.com/kb/en/full-text-index-overview/#in-boolean-mode